lib/ex_aws/operation/polly.ex

```diff
@@ -0,0 +1,57 @@
+defmodule ExAws.Operation.Polly do
```

Comment:
Operations are not per service, but rather per category of service. Polly is a `RestJSON` service. There is not at the moment a RestJSON operation although looking at what we have here I'm pretty sure you could just re-use the JSON operation.

All of the path logic you have that is custom to polly you could just do in the `defp request` function you have inside `ExAws.Polly`